vars/defaultIntegPipeline.groovy

```diff
@@ -76,7 +104,11 @@ def call(Map config = [:]) {
                                     if (skipCaptureProxyOnNodeSetup) {
                                         baseCommand += " --skip-capture-proxy"
                                     }
-                                    sh baseCommand
+                                    withCredentials([string(credentialsId: 'migrations-test-account-id', variable: 'MIGRATIONS_TEST_ACCOUNT_ID')]) {
+                                        withAWS(role: 'JenkinsDeploymentRole', roleAccount: "${MIGRATIONS_TEST_ACCOUNT_ID}", duration: 5400, roleSessionName: 'jenkins-session') {
```

Comment:
Can we tweak down this duration, what is this tied to? 90 minutes is a long time, and its different than the time at line 137 of 60 minutes